satpy/etc/readers/abi_l2_nc.yaml

```diff
@@ -0,0 +1,469 @@
+# GOES-R PUG vol5 L2+ products
+#  https://www.goes-r.gov/products/docs/PUG-L2+-vol5.pdf
+reader:
+  description: NetCDF4 reader for the GOES-R Level 2+ products from NOAA
+  name: abi_l2_nc
+  sensors: ['abi']
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+
+
+datasets:
+  lon:
+    name: lon
+    file_type:
+      - abi_l2_rsr
+      - abi_l2_dsr
+    units: "degrees"
+    standard_name: longitude
+
+  lat:
+    name: lat
+    file_type:
+      - abi_l2_rsr
+      - abi_l2_dsr
+    units: "degrees"
```

Comment:
The units for longitude and latitude should be "degrees_east" and "degrees_north" to follow CF standards.